server/pulp/plugins/file/distributor.py

```diff
@@ -18,6 +18,8 @@
 
 class FileDistributor(Distributor):
     """
+    DEPRECATED: please use the model_distributor.FileDistributor
```

Comment:
What do you think about in logged `DeprecationWarning` or `PendingDeprecationWarning` into the `__init__` of `FileDistributor`. That way, users relying on code that will be removed can raise the issue with the plugin developers?